pkg/ibmcsidriver/controller_helper.go

```diff
@@ -179,6 +179,8 @@ func getVolumeParameters(logger *zap.Logger, req *csi.CreateVolumeRequest, confi
 			}
 		case IsENIEnabled:
 			err = setISENIEnabled(volume, key, strings.ToLower(value))
+		case IsEITEnabled:
+			err = setISEITEnabled(volume, key, strings.ToLower(value))
```

Comment:
Change the method name to checkAndSet for ENI and EIT

Reply:
Updated